projects/sport-data-valley/manual_seeds/helpers/people_helper.rb

```diff
@@ -0,0 +1,21 @@
+def find_or_create_person(email)
```

Comment:
Can you put this in a module, and then just include the module from another module?

It's not very Ruby-like (and error prone) to have all these things living in the global namespace.